src/SqlSquared/Signature/Relation.purs

```diff
@@ -36,7 +36,7 @@ type VariRelR =
   }
 
 type TableRelR =
-  { path ∷ Pt.AnyFilePath
+  { path ∷ Pt.AnyFile
```

Comment:
Would you mind make this `Either AnyFile AnyPath` or do you think this is matter of other pr? 
E.g. marklogic directory querying

Reply:
I think other pr would be better for this :d